src/ArrOperationTest.java

```diff
@@ -0,0 +1,40 @@
+import static org.junit.Assert.*;
+
+import org.junit.Before;
+import org.junit.Test;
+public class ArrOperationTest
```

Comment:
Missing negative test method for all method.

Reply:
Added Negative test cases for all methods